plugins/additional/additional.cpp

```diff
@@ -229,13 +229,21 @@ class RichTextCtrlComponent : public ComponentBase
 {
 public:
 	wxObject* Create(IObject* obj, wxObject* parent) override {
+
+		wxString text = obj->GetPropertyAsString(_("value"));
+
 		wxRichTextCtrl* richText = new wxRichTextCtrl( 	(wxWindow*)parent,
 															wxID_ANY,
-															wxEmptyString,
+															text,
 															obj->GetPropertyAsPoint(_("pos")),
 															obj->GetPropertyAsSize(_("size")),
 															obj->GetPropertyAsInteger(_("style")) | obj->GetPropertyAsInteger(_("window_style")));
 
+		if(!text.IsEmpty())
```

Comment:
Please don't use wxWidgets API if there is std::string API available, here i would prefer to use `empty()`. Also add a space between the `if` and the parentheses.